components/function-controller/internal/controllers/serverless/build_resources.go

```diff
@@ -4,21 +4,18 @@ import (
 	"fmt"
 	"strings"
 
+	appsv1 "k8s.io/api/apps/v1"
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
-
-	"knative.dev/serving/pkg/apis/autoscaling"
+	"k8s.io/apimachinery/pkg/util/intstr"
 
 	serverlessv1alpha1 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
 )
 
 const (
-	servingKnativeVisibilityLabel      = "serving.knative.dev/visibility"
-	servingKnativeVisibilityLabelValue = "cluster-local"
-
-	destinationArg = "--destination"
+	destinationArg        = "--destination"
+	functionContainerName = "function"
```

Comment:
this change means that we'll have to update this container name in UI, we need this in order for logging UI to work

Reply:
I will add this change in https://github.com/kyma-project/console/pull/1822 :)

Reply:
Added https://github.com/kyma-project/console/pull/1822/commits/5b96e746d6f7d70e50accc7a55459f7a10ca9c7b